test/e2e/framework/networking_utils.go

```diff
@@ -69,15 +69,15 @@ var NetexecImageName = imageutils.GetE2EImage(imageutils.Netexec)
 
 // NewNetworkingTestConfig creates and sets up a new test config helper.
 func NewNetworkingTestConfig(f *Framework) *NetworkingTestConfig {
-	config := &NetworkingTestConfig{f: f, Namespace: f.Namespace.Name}
+	config := &NetworkingTestConfig{f: f, Namespace: f.Namespace.Name, HostNetwork: true}
```

Comment:
This looks different than https://github.com/kubernetes/kubernetes/pull/69525, why are we now defaulting `HostNetwork` to `true`?

Reply:
Will fix

Reply:
Indeed. The PR you are mentioning sets the ``HostNetwork`` to ``false`` for the tests in question, but consensus has not been reached in that regards.So, this basically just maintains the default ``HostNetwork: true`` behaviour for the regular tests, while the Windows networking related tests will call NewCoreNetworkingTestConfig with ``hostNetwork = false``.